packages/shared/src/lib/growthbook/index.ts

```diff
@@ -0,0 +1,3 @@
+export type { Context } from './types/growthbook';
```

Comment:
Maybe we can use [pnpm patch](https://pnpm.io/cli/patch) to remove all the garbage we don't need from their SDK. This way we will not need to add that code to our codebase and it would still load through node modules.

Reply:
Good idea. I'll see what can be improved :)

Reply:
At the end I forked the project, removed whatever needed and used pnpm to force using my fork. @capJavert wdyt?

Reply:
Sounds good also, we just need to be aware of maintaining that when they make updates 👍